src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -138,6 +138,10 @@ func runServerProxy(cfg *config.Config) {
 			ServerCertFile: config.InConfigDir("servercert.pem"),
 		},
 		AllowedPorts: []int{80, 443, 8080, 8443, 5222, 5223, 5228},
+
+		// We allow all censored countries plus us, es and mx because we do work
+		// and testing from those countries.
+		AllowedCountries: []string{"US", "ES", "MX", "CN", "VN", "IN", "IQ", "IR", "CU", "SY", "SA", "BH", "ET", "ER", "UZ", "TM", "PK", "TR", "VE"},
```

Comment:
List of censored countries taken from here - https://github.com/getlantern/lantern/blob/devel/src/main/java/org/lantern/DefaultCensored.java#L33